addons/accessibility/fnc_shouldPlay.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_accessibility_fnc_shouldPlay
+
+Description:
+    Checks if a sound should be played based on the categories it is in.
```

Comment:
It only checks the cats. It doesn't handle sounds at all :)

"Check if all of the given sound categories are enabled."